ArduPlane/defines.h

```diff
@@ -48,7 +48,7 @@ enum failsafe_action_long {
 enum class StickMixing {
     NONE     = 0,
     FBW      = 1,
-    DIRECT   = 2,
+    DIRECT_REMOVED  = 2,
```

Comment:
renaming this to "DIRECT_REMOVED" seems odd. Why not just RESERVED" /w a comment?

Reply:
I mean I could rename it to `RESERVED` and add a comment `Direct (removed)`, there is a big comment explaining in the one place were its used.

Reply:
Wouldn't it break if later another value was removed?

Reply:
I don't understand the question, maybe you can explain a little more?